migrations/2023-12-12-150828_community_aggregates_add_local_subscribers/up.sql

```diff
@@ -0,0 +1,70 @@
+-- Couldn't find a way to put local_subscribers right after subscribers except recreating the table.
+ALTER TABLE community_aggregates
+    ADD COLUMN local_subscribers int8 NOT NULL DEFAULT 0;
+
+-- update initial value
+UPDATE
+    community_aggregates
+SET
+    local_subscribers = (
+        SELECT
+            COUNT(*)
+        FROM
+            community_follower cf
+        WHERE
+            cf.community_id = community_aggregates.community_id
+            AND (
+                SELECT
+                    local
+                FROM
+                    person
+                WHERE
+                    person.id = cf.person_id));
+
+CREATE OR REPLACE FUNCTION community_aggregates_local_subscriber_count ()
```

Comment:
No point in creating a new function, just add this block below the existing one in `community_aggregates_subscriber_count` . 

Or if you really want to keep them separate, make sure the trigger runs both functions.